sample.env

```diff
@@ -202,6 +204,8 @@ CM_STARTUP_DELAY_SECONDS=90
 INTERSECTION_API_ENABLE_API=true
 INTERSECTION_API_ENABLE_EMAILER=true
 INTERSECTION_API_ENABLE_REPORTS=true
+KEYCLOAK_CLIENT_API_ID=cvmanager-api
```

Comment:
I don't see that we are using these variables - everything should be referencing KEYCLOAK_API_CLIENT_ID and KEYCLOAK_API_CLIENT_SECRET_KEY
For running locally, I usually use a application-dev.yaml